pypeclub/OpenPype
# Photoshop: synchronize image version with workfile

## Brief description
Adds configurable Setting to control if publishable instances ('image', 'review' family) should take version from workfile.



## Testing notes:
1. configure `project_settings/photoshop/publish/CollectVersion`
![Screenshot 2022-09-16 143114](https://user-images.githubusercontent.com/4457962/190639289-a5ee3f10-1cf2-4aa1-98ed-3413e3f41d3b.png)


3. publish only workfile (disable image and review instances) - that way gap in their versions will be created
4. publish from Photoshop (manually and from WP)
5. version of image (and review) should match to workfile
